elf/elf.go

```diff
@@ -257,7 +322,21 @@ func elfReadMaps(file *elf.File) (map[string]*Map, error) {
 			mapCount := len(data) / C.sizeof_struct_bpf_map_def
```

Comment:
For the record, the idea to have one ELF section per map comes from:
https://github.com/safchain/goebpf/blob/b1e37d587ca2eb6a589c72efd7d8306372d3e491/goebpf.go#L469
It's easier because then it does not need to get the name from the symbol table.